erpnext/www/lms/program.py

```diff
@@ -26,4 +27,14 @@ def get_program(program_name):
 
 def get_course_progress(courses, program):
 	progress = {course.name: utils.get_course_progress(course, program) for course in courses}
-	return progress
\ No newline at end of file
+	return progress
+
+def calculate_progress(progress):
```

Comment:
Is this the same function as the one in course.py? Why is it being written twice? It could just be imported and called here

Reply:
@rvpasari No.
the calculate_progress() function is not same as course.py's function.
by mistake, I had given the same function name for calculating the progress of courses/topics/topics_content but the operation which is done under this function is different for each progress like course_progress,topic_progress and topic_content_rpogress.

In my latest commit, you can see a proper function name for each progress.